samples/name_domains_values/Coded_Domain_METOC_Space_Point_Entities.csv

```diff
@@ -1 +0,0 @@
-Name,Value
```

Comment:
Since the METOC Space symbol set is empty, there will be no domain file or subtype metadata produced by JMSML for this particular symbol set.